Dockerfile

```diff
@@ -3,14 +3,21 @@ FROM docker.io/centos:centos7
 RUN yum install -y python-requests && \
     curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b stein current-tripleo && \
     yum update -y && \
-    yum install -y openstack-ironic-api openstack-ironic-conductor crudini \
-        iproute iptables dnsmasq httpd qemu-img-ev iscsi-initiator-utils parted gdisk ipxe-bootimgs psmisc sysvinit-tools \
-        mariadb-server python-PyMySQL python2-chardet && \
+    yum install -y crudini iproute iptables dnsmasq httpd qemu-img-ev \
+        iscsi-initiator-utils parted gdisk ipxe-bootimgs psmisc \
+        sysvinit-tools mariadb-server python-PyMySQL python2-chardet \
+        python-pip git gcc python-devel && \
```

Comment:
ipmitool?

Reply:
also pip requires epel, we cannot just install it..

Reply:
I think pip doesn't require epel downstream, so we should be good

Reply:
I'm pretty sure RHEL did not have pip the last time I checked. Maybe it has changed..

Reply:
Hmm, you're right, apparently it was added in RHEL 8. CentOS 7 is still an issue, I wonder if tripleo-repos enabled EPEL.

Reply:
It would not surprise me if tripleo-repos enables EPEL since it is the only place I could find some packages when I was looking at a partial rpm + source approach.

Reply:
tripleo repos If I remember correctly do not enable epel but instead rely on RDO's RPM builds of packages for CI.